packages/wallet-service/src/api/wallet.ts

```diff
@@ -380,6 +381,63 @@ interface LoadResult {
   xpubkey: string;
 }
 
+/*
+ * This lambda will be started by a SNSMessage on the load failed SNS configured
+ * in serverless.yml. It will receive all wallet load failed events published on
+ * the loadWalletAsync DLQ SNS topic
```

Comment:
It would be interesting to have here an example of the event received.

Reply:
Done in 33f81483e042dae9bc6a820a8eca52c8213a74b7